src/entities/Place/model.ts

```diff
@@ -186,6 +187,15 @@ export default class PlaceModel extends Model<PlaceAttributes> {
           UserLikesModel
         )} ul on p.id = ul.place_id AND ul."user" = ${options.user}`
       )}
+      ${conditional(
+        !!options.categories.length,
+        SQL`INNER JOIN ${table(
```

Comment:
was this query checked?

Reply:
what do you mean? check its performance or is it working as expected?